src/operun/crm/ldap/ldap_sync.py

```diff
@@ -204,49 +299,41 @@ def generate_update_mod_attrs(self, obj):
         mod_attrs = self.generate_mod_attrs(obj)
         return [(ldap.MOD_REPLACE,) + item for item in mod_attrs]
 
-    def generate_ldap_node(self, obj):
+    def generate_ldap_dn(self, obj, cn=False, superior=False):
```

Comment:
`def generate_ldap_dn(self, obj)
    cn = self.generate_ldap_cn(obj)
    superior = self.generate_ldap_superior(obj)
    dn = cn + ',' + superior
    return dn

def generate_ldap_cn(self, obj)
    cn = 'cn={0}'.format(obj.Title())     
    return cn

def generate_ldap_superior(self, obj)
    # code as we have it
`